app/assets/stylesheets/partials/footer.sass

```diff
@@ -0,0 +1,4 @@
+.footer
+  margin-top: 5rem
+  border-top: 1px solid #e5e5e5
+  padding-top: 1rem
```

Comment:
could we:
- add `color`: `$gray` (deemphasize it a little)
- add `padding-bottom` of 1rem (right now looks a little truncated with pages with height >= 1 window height)

thanks!